Script/World.gd

```diff
@@ -0,0 +1,93 @@
+extends Node2D
+
+@export var level_directory: String
```

Comment:
It's annoying to have to set this on each scene. Unfortunately there doesn't seem to be a way to, at runtime, ask for the path of the current scene.

It's also annoying that Godot doesn't seem to have an `@export_file` variation for directories. This could be made to be a list of levels but I think being able to drop numbered levels into a directory is good...

Reply:
Actually there is a way: `Node.scene_file_path`! I'll try using this tomorrow.